binr/rabin2/rabin2.c

```diff
@@ -188,8 +188,11 @@ static bool extract_binobj(const RBinFile *bf, RBinXtrData *data, int idx) {
 	if (!path) {
 		return false;
 	}
-	// XXX: Wrong for w32 (/)
 	ptr = strrchr (path, DIRSEP);
+#if __WINDOWS__
+	char *p = strrchr (path, '/');
```

Comment:
strrchr is not portable, better use r_str_rchr

Reply:
also this, with the other patch from yesterday seems like workarounds to me, we should probably use or implement more generic filesystem apis

Reply:
yeah, didn't notice there was already one api for that.